PegSolitaire.h

```diff
@@ -0,0 +1,30 @@
+#ifndef DEMO_PEGSOLITARE_H
+#define DEMO_PEGSOLITARE_H
+
+#include <array>
+
```

Comment:
Manjka #include <string>
Dobro je da header vsebuje vse include, ki se rabijo, pa tudi brez tega se mi ne zbuilda.

Reply:
A se premaknejo vsi inkludi tudi v header file iz cpp fajla? Čeprav so porabljeni samo zaradi implementacije

Reply:
Ne. V header gre samo to kar se v headerju rabi.